### PR TITLE
[luci-interpreter] Introduce Abs kernel

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This commit adds Abs kernel to luci-interpreter.

for issue: #10185

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

**Review by AShedko — DISMISSED** [review]
LGTM

File: compiler/luci-interpreter/src/kernels/Abs.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      break;
+    case DataType::S8:
+      eval<int8_t>();
+      break;
+    case DataType::U8:
+      eval<uint8_t>();
+      break;
+    case DataType::S16:
+      eval<int16_t>();
+      break;
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Abs.cpp:53]: There are some issues in integer types.

- s32: Size of `int` differs across platforms.
- u8, s8, s16: Quantized value. These types should be handled differently.

Please remove them and only leave FLOAT32. FLOAT32 is enough for quantization.

File: compiler/luci-interpreter/src/kernels/Abs.cpp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/Abs.h"
+
+#include "kernels/Utils.h"
+
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Abs.cpp:20]:
```suggestion
#include <cmath> // abs for float

```

File: compiler/luci-interpreter/src/kernels/Abs.cpp (hunk 1 above)
Comment (BalyshevArtem) [reply compiler/luci-interpreter/src/kernels/Abs.cpp]: done

**Review by jinevening — APPROVED** [review]
LGTM

**Review by AShedko — APPROVED** [review]

**Pull request merged**